library/Icingadb/ProvidedHook/RedisHealth.php

```diff
@@ -28,6 +28,16 @@ public function checkHealth()
             }
 
             $instance = Instance::on($this->getDb())->columns('heartbeat')->first();
+
+            if ($instance === null) {
+                $this->setState(self::STATE_UNKNOWN);
+                $this->setMessage(t(
+                    "Can't connect to Icinga Redis: Icinga DB is not running or not writing into 
```

Comment:
The connection to redis has been already attempted. Use "Can't check Icinga Redis"